src/sif/autocost.cc

```diff
@@ -432,9 +448,10 @@ bool AutoCost::AllowedReverse(const baldr::DirectedEdge* edge,
 // Get the cost to traverse the edge in seconds
 Cost AutoCost::EdgeCost(const baldr::DirectedEdge* edge,
                         const graph_tile_ptr& tile,
-                        const uint32_t seconds) const {
+                        const uint32_t seconds,
+                        uint8_t& flow_sources) const {
   // either the computed edge speed or optional top_speed
-  auto edge_speed = tile->GetSpeed(edge, flow_mask_, seconds);
+  auto edge_speed = tile->GetSpeed(edge, flow_mask_, seconds, false, &flow_sources);
```

Comment:
Adding `&flow_sources` here allows us to save off which speed sources were used in this speed calculation.  This is flattened down to a single bit on the edgelabel.  Which basically indicates if we have some sort of traffic data for this edge.

Reply:
Any reason why this isn't a pointer like the equivalent param in `GraphTile::GetSpeed()`? Pointer is good if not everyone is expected to use this param (eg in `DynamicCost::EdgeCost` we're creating an unused var because of the need for pass by ref).